integration/integration_test.go

```diff
@@ -7338,17 +7339,26 @@ func testAgentlessConnection(t *testing.T, suite *integrationTestSuite) {
 		Cluster:   helpers.Site,
 	}, tc.Username)
 	require.NoError(t, err)
+	t.Cleanup(func() {
+		// the error is ignored here to avoid failing on io.EOF
+		_ = nodeClient.Close()
+	})
 
 	// forward SSH agent
 	sshClient := nodeClient.Client.Client
 	session, err := sshClient.NewSession()
 	require.NoError(t, err)
 	t.Cleanup(func() {
-		require.NoError(t, session.Close())
+		// the error is ignored here to avoid failing on io.EOF
```

Comment:
Why did we move from having no error to returning EOF sometimes?

Reply:
Because the ssh server from `startSSHServer` now closes the session to avoid a deadlock, I'll comment that

Reply:
Done